earth_enterprise/src/server/geresetpgdb

```diff
@@ -201,27 +207,49 @@ if [ -e $VARPGDIR/data/postmaster.pid ]; then
 fi
 
 if [ $type == "backup" ]; then
-  if [ ! -e $VARPGDIR/data ]; then
-    echo "WARNING: There is no data to back up."
```

Comment:
Any reason that info about no data to back up was removed ? It would be, still, useful to post info to a user if they asked for backup and there was nothing to back up.

Reply:
The check is no longer valid as the directory will always be there after the unbundle (this change is in anticipation of that). If we need to, we can do a different check. I'm not entirely sure it's necessary.

Reply:
ok